snippets/es6.cson

```diff
@@ -0,0 +1,74 @@
+'.source.coffee':
+
+  'Ember Snippets for es6 syntax':
+    'prefix': 'es6--help'
+    'body':
+      """
+      ${1:
+        # snippets to generate --ES6 syntax--
+        # example: imp-ds<press tab>
+        #  ___________________________________________________________
+        # | type short (or)  long version    ->   then press <tab>    |
+        # | ----------  ->  ---------------  -> ----------------------|
+        # |  imp-ds     ->   import-ds       ->  `import DS...      ` |
+        # |  imp-emb    ->   import-ember    ->  `import Ember...   ` |
+        # |  exp-def    ->   export-default  ->  `export default... ` |
+        # |  es6        ->   es6             ->  - full set -         |
+        # |___________________________________________________________|
+        }
+      """
+
+  'ES6 import Ember':
+    'prefix': 'import-ember'
```

Comment:
They are meant to be used in case of when the user forgets or suspects what to type.
I think it is annoying to see them every time,while they are already have got stock in your mind.